src/object/cli_obj.c

```diff
@@ -2516,16 +2987,15 @@ obj_comp_cb(tse_task_t *task, void *data)
 				obj_auxi->io_retry = 0;
 			}
 		}
-		if (!obj_auxi->spec_shard && task->dt_result == -DER_INPROGRESS)
-			obj_auxi->to_leader = 1;
 	}
 
 	if (pm_stale || obj_auxi->io_retry)
 		obj_retry_cb(task, obj, obj_auxi, pm_stale);
 
 	if (!obj_auxi->io_retry) {
 		struct obj_ec_fail_info	*fail_info;
-		bool new_tgt_fail = false;
+	 	bool new_tgt_fail = false;
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs